miniapps/navier/navier_tgv.cpp

```diff
@@ -0,0 +1,401 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#include "navier_solver.hpp"
+#include <fstream>
+
+using namespace mfem;
+using namespace navier;
+
+struct s_NavierContext
+{
+   int ser_ref_levels = 1;
+   int order = 4;
+   double kinvis = 1.0 / 1600.0;
+   double t_final = 10 * 1e-3;
+   double dt = 1e-3;
+   bool pa = true;
+   bool ni = false;
+   bool visualization = false;
+   bool checkres = false;
+} ctx;
```

Comment:
I'd like to keep it that way. I did it that way because there was a configuration system in the public version before, that is now only existing in an internal project.